test/test_list.rb

```diff
@@ -568,3 +568,54 @@ def test_position_after_moving_to_empty_list
     assert_equal [1], ArrayScopeListMixin.where(:parent_id => 4, :parent_type => 'anything').order(:pos).map(&:pos)
   end
 end
+
+class TouchTest < ActsAsListTestCase
+  def setup
+    setup_db
+    4.times { ListMixin.create! updated_at: 1.day.ago }
+  end
+
+  def test_moving_item_lower_touches_self_and_lower_item
+    ListMixin.first.move_lower
+    ListMixin.pluck(:updated_at)[0..1].each do |updated_at|
+      assert_in_delta updated_at, Time.now.utc, 1.second
+    end
+    ListMixin.pluck(:updated_at)[2..3].each do |updated_at|
+      assert_in_delta updated_at, 1.day.ago, 1.second
+    end
+  end
+
+  def test_moving_item_higher_touches_self_and_higher_item
```

Comment:
Assignment Branch Condition size for test_moving_item_higher_touches_self_and_higher_item is too high. [17/15]